tests/foreman/api/test_registration.py

```diff
@@ -88,9 +87,10 @@ def test_host_registration_end_to_end(
     assert CLIENT_PORT == rhel_contenthost.subscription_config['server']['port']
 
 
+@pytest.mark.no_containers
 @pytest.mark.tier3
 def test_positive_allow_reregistration_when_dmi_uuid_changed(
-    module_org, rhel_contenthost, target_sat
+    module_org, rhel_contenthost, target_sat, module_ak_with_synced_repo, module_location
```

Comment:
Can we use module_target_sat instead?

Reply:
What difference do you see in module_target_sat and target_sat?
If I am not missing something, they are pointing to the same Satellite object in the end.

Reply:
I would prefer `target_sat` here as we are changing dmi_uuid of the satellite and also doesn't make difference to use `module_target_sat` here.

Reply:
Other tests in this module are using module_target_sat, so ask was just to keep use of it consistent